Makefile

```diff
@@ -636,7 +636,7 @@ run-android-ui-test-spoon:
 
 .PHONY: apackage
 apackage:
-	cd platform/android && ./gradlew --parallel --max-workers=$(JOBS) assemble$(BUILDTYPE)
+	if [ -d $(“platform/android/MapboxGLAndroidSDK/src/main/jniLibs”) ]; then cd platform/android && ./gradlew --parallel --max-workers=$(JOBS) assemble$(BUILDTYPE); fi	
```

Comment:
This fix was mostly for CI integration where we always start with a clean docker image. Any ideas how to achieve this more cleanly?